ui/src/pages/version/components/forms/components/transformer/InputPanel.js

```diff
@@ -11,16 +11,20 @@ import {
 import { get, useOnChangeHandler } from "@gojek/mlp-ui";
 import { AddButton } from "./components/AddButton";
 import { TableInputCard } from "./components/table_inputs/TableInputCard";
+import { AutoloadCard } from "./components/table_inputs/AutoloadCard";
 import { VariablesInputCard } from "./components/table_inputs/VariablesInputCard";
 import { FeastInputGroup } from "../feast_config/components/FeastInputGroup";
 import { EncodersInputGroup } from "./components/table_inputs/EncodersInputGroup";
 import { Panel } from "../Panel";
 import {
+  Autoload,
   FeastInput,
   TablesInput
 } from "../../../../../../services/transformer/TransformerConfig";
 
-export const InputPanel = ({ inputs = [], onChangeHandler, errors = {} }) => {
+import {PROTOCOL} from "../../../../../../services/version_endpoint/VersionEndpoint"
```

Comment:
nit: there should be `;` by the end of this line. seems like the linter is not working correctly on your machine.